GCM/App/init/ScenesInitializerImpl.java

```diff
@@ -38,6 +44,29 @@ private void populateInitializersMap() {
 	@Override
 	public EnumMap<SceneNames, Scene> getScenes() { return scenes; }
 	
+<<<<<<< HEAD
+	private void populateFxmlFilesMap(){
```

Comment:
Again, the "<<<<<<" part.

Btw you need to do another pull because I completely changed this class